rfcs/2021-19-02-units-for-typography-and-layout.md

```diff
@@ -0,0 +1,603 @@
+   * Yes. we'll store things as unitless in tokens
+3. Couldn't find what does WCAG says about this exactly? For eg: it just says the content should be readable at 200% zoom - [WCAG Guideline](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html)
+4. React Native equivalent for rem?
+   * There's no equivalent. React Native is unitless and everything is considerd as `dp`
+5. The layout and typography scale for web and react native has to be differnt?
+   * Probably yes. Because resolution on mobile are becoming denser and the area is still the same which means if we keep things on mobile comparatively bigger to it's desktop counterparts it'll yield better results. [Spectrum](https://spectrum.adobe.com/page/platform-scale/#Proportions) did a user reasearch and came up with desktop to mobile ratio of `1:1.25` i.e on mobile, things are 25% bigger compared to desktop.
+
+# References
+* Accessible font Sizing - [CSS Tricks](https://css-tricks.com/accessible-font-sizing-explained/)
+* Absolute vs Relative units - [24a11y](https://www.24a11y.com/2019/pixels-vs-relative-units-in-css-why-its-still-a-big-deal/)
+* Discussion threads on Twitter which has thoughts from different people for different use cases and scenarios
+  1. https://twitter.com/_kamlesh_/status/1361264818821468163
+  2. https://twitter.com/threepointone/status/1362348163689033732
+  3. https://twitter.com/naman34/status/1362626358539481090
+
+* Scaling issues - https://twitter.com/_kamlesh_/status/1362384908375584774
+* POC to visualise how things are different for different units on web - https://twitter.com/_kamlesh_/status/1362350508242042884
+* [Designing for a new generation of mobile devices](https://juiceboxinteractive.com/blog/a-pixel-is-not-a-pixel-designing-for-a-new-generation-of-mobile-devices/)
+* [Compare device's width, height, ppi](https://www.mydevice.io/#compare-devices)
+* [How device pixel ratio is calculated for web](https://www.html5rocks.com/en/mobile/high-dpi/)
+* [How device pixel ratio is calculated for native apps](https://www.perfecto.io/blog/how-mobile-screen-size-resolution-and-ppi-screen-affect-test-coverage)
+* [Understanding Device Resolution for Web Design and Development](https://medium.com/@flik185/understanding-device-resolution-for-web-design-and-development-3bb4a5183478)
+* [Pixel density on native apps](https://material.io/design/layout/pixel-density.html)
+* [Designing for multiple screen densities on Android](https://developerlife.com/2018/07/21/designing-for-multiple-screen-densities-on-android/)
+* [Build responsive React Native views for any device and support orientation change](https://medium.com/react-native-training/build-responsive-react-native-views-for-any-device-and-support-orientation-change-1c8beba5bc23)
+* [iPhone Resolutions](https://www.paintcodeapp.com/news/ultimate-guide-to-iphone-resolutions)
+
+# Lesser known facts
```

Comment:
Should we move this section before the detailed discussion? Would be easier for people to follow the reasoning if they don't know these points.